code/lib/builder-webpack5/package.json

```diff
@@ -25,15 +25,15 @@
       "import": "./dist/index.mjs",
       "types": "./dist/index.d.ts"
     },
-    "./presets/custom-webpack-preset.js": {
-      "require": "./presets/custom-webpack-preset.js",
-      "import": "./presets/custom-webpack-preset.mjs",
-      "types": "./presets/custom-webpack-preset.d.ts"
+    "./presets/custom-webpack-preset": {
+      "require": "./dist/presets/custom-webpack-preset.js",
+      "import": "./dist/presets/custom-webpack-preset.mjs",
+      "types": "./dist/presets/custom-webpack-preset.d.ts"
     },
-    "./presets/preview-preset.js": {
-      "require": "./presets/preview-preset.js",
-      "import": "./presets/preview-preset.mjs",
-      "types": "./presets/preview-preset.d.ts"
+    "./presets/preview-preset": {
+      "require": "./dist/presets/preview-preset.js",
+      "import": "./dist/presets/preview-preset.mjs",
+      "types": "./dist/presets/preview-preset.d.ts"
```

Comment:
These export maps were incorrect, pointing to the wrong folder, and using extensions, when we typically do not.